docs/source/diagrams/pbft-view-change.puml

```diff
@@ -0,0 +1,129 @@
+@startuml
+!pragma teoz true
+hide footbox
+
+participant Node1
+participant Node2
+participant Node3
+participant Node4
+
+note over Node1, Node4
+    When nodes have a quorum (n-f) of INSTANCE_CHANGE messages for a same number of view
+    <b>Start a View Change</b>
+end note
+
+
+loop repeat for Node2, Node3, Node4
+    note over Node1, Node4
+        <b>Node1 send VIEW_CHANGE </b>
+    end note
+
+    Node1 -> Node4: VIEW_CHANGE(Node1)
```

Comment:
We are unstashing ViewChange-related messages here (once start a view change to the next view)